### DISCUSSION TITLE
[Floating label] Does floating label work only within menu/item elements?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1541] [Elements (button, image, label, loader, ...)] [answered]
Hello,
I need to put a circular label under an icon, something like this:

![image](https://user-images.githubusercontent.com/5399265/85859364-54585b00-b7bd-11ea-9606-0d01c3f62665.png)

I did it, but I expect to do it in a easiest way, using [floating label](https://fomantic-ui.com/elements/label.html#floating).
Does it work only if it is an `item` inside a `menu`?

https://jsfiddle.net/jk5pnugL

**Comment by lubber-de** [comment, marked as answer]
As mentioned in the docs https://fomantic-ui.com/elements/label.html#floating

> A floating label must be positioned inside a container with position: relative to display properly.

Add `position:relative` to your div wrapper and it works as expected
See https://jsfiddle.net/lubber/teprd9bs/3/

**Comment by smartm0use** [comment]
Oh, sorry, my mad. Seeing menu/item examples only has been misleading for me.
Thank you so much for your support, as usual ;)
